src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -161,17 +169,28 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler {
   /// Send any buffered resource usage as a single publish.
   void SendBatchedResourceUsage();
 
+  /// Prelocked version of GetResourceUsageBatchForBroadcast. This is necessary for need
+  /// the functionality as part of a larger transaction.
+  void GetResourceUsageBatchForBroadcastUnsafe(rpc::ResourceUsageBatchData &buffer)
```

Comment:
```suggestion
  void GetResourceUsageBatchForBroadcast_Locked(rpc::ResourceUsageBatchData &buffer)
```

I think this is the convention